dev-libs/ffcall/files/libffcall-2.5-add-support-for-clang.patch

```diff
@@ -0,0 +1,51 @@
+From fe6d9f089d8326b077420c10fec8c2174557be30 Mon Sep 17 00:00:00 2001
```

Comment:
Always include the link to the upstream patch at the top, please.

Reply:
I am very worried about this patch! It is *older* than 2.5.


... they reverted it. https://cgit.git.savannah.gnu.org/cgit/libffcall.git/commit/?id=35692c0a38c4868f42c884728c23833d8d36e83e